src/data/decimal/decimalFn.ts

```diff
@@ -0,0 +1,72 @@
+import { assert } from '../../utils/assert'
+import { PLDecimal } from './PLDecimal'
+
+export function plDecimal(strValue: string): PLDecimal {
+  const decimalObj = parseNumString(strValue)
+  return new PLDecimal(decimalObj.intValue, decimalObj.decimals)
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function assertNumeric(strValue: any): any {
```

Comment:
1+3) The source of the function is [this](https://stackoverflow.com/questions/175739/built-in-way-in-javascript-to-check-if-a-string-is-a-valid-number#175787) link. As the comment says, `parseFloat` cannot check if the entire string is numeric (e.g. it can parse `0.2` from `0.2hello`). If you have better solution, please let me know.

2) I changed the return type to `boolean` as in the `assert` function.